### ISSUE TITLE
Returning expeditions kill the game

### ISSUE THREAD

**Issue by kekima** [Madmanmayson/KanColleViewer#9]
Heya. Ever since the 4.2.11 update I can't get past the loading of a returning expedition if I run the KCV after one of the expeditions has returned. I'm getting around it by running the older 4.2.10 version to accept all the expeditions and then run 4.2.11 again (4.2.11.1 has same problem) but it really gets annoying. 

I also tried to just move the updated files from 4.2.11 to the 4.2.10 version and somehow it works, but battleinfo plugin isn't showing me routing.

**Comment by Madmanmayson** [comment]
The routing part on BIP is still a little broken (works on some occasions others not) that I've been working on looking into after noticing it on E3P3 as for the expeditions I'll look into it. I wasn't aware it was being cause from launching the program and then having them return so that will help find the issue since I always have it running and didn't notice that.

**Comment by kekima** [comment]
Actually, it appears that it has something to do with updating in general, sorry for misleading you. It got worse, I can't even get out of the refuel screen anymore. I returned from 4-3 farm, refueled my ships but KCV was still showing me that they aren't. When I went to refuel them again, game went into perpetual loading again. I refreshed it, decided to go for another 4-3 run but it went into the perpetual loading as soon as I clicked the red sortie banner.

**Comment by Madmanmayson** [comment]
The issue with 4.2.10 is that post DMM maintenance, it has tendencies to have connections issues to the site itself promting the upstreamed update to 4.2.11. Now since it's caused so many issues in the past and I still have no real idea where, try removing the Calculator.dll and see if that helps

**Comment by kekima** [comment]
Hey, neat. I think it's working. Welcomed my shipgirls without any trouble. I guess Calculator was the problem.

**Comment by Madmanmayson** [comment]
Yeah unfortunately it's an issue that everyone but me has making it really hard to find the cause in the plugin.

**Comment by Madmanmayson** [comment]
Also another note as for the nodes/route not being updated. For now you can delete the BattleInfoPluginData folder and it should fix it until you change your fleet comp for that map (currently testing a hopeful fix for the plugin itself)
